### PR TITLE
Make access open for mocking/subclassing

### PR DESCRIPTION
**Pull request by jasonm23**

### CODE REVIEW

**Comment by jasonm23** [comment]
@Econa77  It looks like Danger needs more permissions to post comments.

**Comment by jasonm23** [comment]
```
bundler: failed to load command: danger (/Users/runner/work/Sauce/Sauce/vendor/bundle/ruby/3.2.0/bin/danger)
```

**Comment by jasonm23** [comment]
Possible token issue for Danger

My downstream build is passing, might be a cache issue.

I'm not sure.

**Comment by Econa77** [comment]
@jasonm23 It seems that the default permission of `GITHUB_TOKEN` of PullRequest from the fork repository is read only.
I tried changing the permissions, so could you please try rebasing?

**Comment by jasonm23** [comment]
should be ok now, thanks

**Comment by Econa77** [comment]
https://github.com/Clipy/Sauce/actions/runs/6459808996/job/17536562328?pr=56#step:1:18

It still looked like `read`. I'll investigate a little more.

**Comment by Econa77** [comment]
@jasonm23 We have temporarily disabled Danger because we were unable to run Danger correctly with the current token ( #62 ). Could you please rebase it again?

**Comment by jasonm23** [comment]
Rebased, thank you

File: .github/workflows/Danger.yml
Comment (Econa77) [inline .github/workflows/Danger.yml:1]: @jasonm23 Thanks 👍 Could you please remove the changes to this file?

Comment (jasonm23) [reply .github/workflows/Danger.yml:1]: removed all changes from Danger.yml

**Review by Econa77 — APPROVED** [review]
👍

**Comment by jasonm23** [comment]
Thank you.

**Comment by jasonm23** [comment]
@Econa77 can you schedule a tag / podspec update please.

**Comment by Econa77** [comment]
@jasonm23 Sorry for the delay. This change has been released in `v2.4.1`.

**Comment by jasonm23** [comment]
Thank you!

**Pull request merged**